packages/manager/src/queries/account.ts

```diff
@@ -69,6 +74,42 @@ export const useChildAccounts = ({
   );
 };
 
+export const useChildAccountsInfiniteQuery = ({
+  filter,
+  headers,
+  params,
+}: RequestOptions) => {
+  const { data: profile } = useProfile();
+  const { data: grants } = useGrants();
+  const hasExplicitAuthToken = Boolean(headers?.Authorization);
+
+  return useInfiniteQuery<ResourcePage<Account>, APIError[]>(
+    [queryKey, 'childAccounts', 'paginated', params, filter],
```

Comment:
We should probably make the query key different than the paginated query to prevent them overwriting each other

We also should consider putting `headers` in the query key. 

```suggestion
    [queryKey, 'childAccounts', 'infinite', params, filter],
```

Reply:
@bnussman-akamai yes definitely missed this - good catch